data/containers/htpasswd/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM registry.opensuse.org/opensuse/bci/golang:latest as builder
+
+COPY main.go /go/src/htpasswd/
+COPY go.* /go/src/htpasswd/
+
+RUN go build -C src/htpasswd/
```

Comment:
Not sure about the work dir for `go` here: should we move to `/go` ?

Reply:
> Not sure about the work dir for `go` here: should we move to `/go` ?

It's `/go/src` but I changed the code to use `WORKDIR` to specify this directory only once.